MAINTAINERS.md

```diff
@@ -0,0 +1,197 @@
+by one of the existing maintainers or regular contributors. C4Ms should specify
+if the call is for a General Maintainer, or for a Scoped Maintainer. If the
+role is for a Scoped Maintainer, the C4M should describe the need for a new
+maintainer, and link to some Pull Requests (or other material) that would be
+relevant for such a maintainer to be responsible for.
+
+Following the C4M, anyone may respond to the issue with a nomination for the
+position and a description of their qualifications. Self-nomination is
+encouraged. Nominations must be accepted. After 2 weeks from the date of issue
+creation, assuming there is at least 1 nomination, the maintainers may select
+one or more candidate from the nominee pool at their discretion. The selected
+candidate should create a PR for adding their key to the repository and to the
+above table (see https://github.com/bitcoin/bitcoin/pull/23798 for example).
+During / Before the PR, regular contributors are encouraged to correspond with
+candidate(s) publicly about any concerns or questions. Candidates are also
+encouraged to write up a Prioritization statement (see below) in conjunction
+with the key PR.
+
+At the discretion of the maintainers, and no sooner than 2 weeks after the PR
+is opened, the candidate(s) PR may be merged.
+
+
+## Maintainer Prioritizations
+
+While the maintainers are ultimately not accountable to any particular roadmap,
+nor can make regular contributors focus on one area or another, practically the
+views and prioritizations of maintainers have an impact on what gets reviewed,
+merged, and the general direction of the project.
+
+As such, maintainers should regularly deliver a written "state of the union"
+article on what they intend to focus on in the coming period. Such addresses
+should be done per-release cycle, and should minimally cover:
+
+1. What went well in the last release cycle
+1. What is making good progress for this release cycle
+1. How the maintainer intends to prioritize outstanding
+   PRs/issues/features/projects in the next release cycle
+1. Projects which could use participation from other contributors
+1. Longer term focuses beyond the next release
+
+Maintainers are generally free to structure their posts as they like. Posts are
+not expected to be "all new content", they can be simple updates from the last
+post. Posts may also be more frequent, if there is significant updates to share.
+
+Historical examples of similar postings:
+
+- [@laanwj](https://laanwj.github.io/2021/01/21/decentralize.html)
+- [@achow101](https://achow101.com/2020/10/0.21-wallets)
+
+
+Below, each maintainer should keep up-to-date links of current and past
+prioritization addresses (as well as any other useful materials in that
+respect).
+
+#### @achow101
+#### @fanquake
+#### @hebasto
+#### @laanwj
+#### @marcofalke
+#### @sipa
+
+Non maintainers may also share similar documents detailing their work here:
```

Comment:
I don't think we want to use this repository as a world-writeable blog for anyone (or a link collection to such)